From 20068a792a7a1447472bf6810561f598ba15e0da Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Mon, 20 Oct 2014 14:03:28 +0800
Subject: [PATCH] crt0: initial clock through c code

As the boot code has been relocated to sram, so configure clock
in sram directly.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
---
 crt0_gnu.S | 1 -
 1 file changed, 1 deletion(-)

diff --git a/crt0_gnu.S b/crt0_gnu.S
index bd00d2a..1bcb38c 100644
--- a/crt0_gnu.S
+++ b/crt0_gnu.S
@@ -109,7 +109,6 @@ _relocate_to_sram:
 	ldrcc   r2, [r1], #4
 	strcc   r2, [r3], #4
 	bcc     1b
-	ldr	pc, =_setup_clocks
 #endif /* CONFIG_FLASH */
 
 	ldr     r4, = lowlevel_clock_init
-- 
1.9.1

